### PR TITLE
changelog and version updates for version 1.1 release

### PR DESCRIPTION
**Pull request by brittonsmith**
The wording in the changelog is from the release notes.

@neutrinoceros, @cphyc, @willsonho2000, as new contributors, I'd like to add your names to the [citable entry on zenodo](https://zenodo.org/record/1458961#.YbEKpvHP3xU). If you would like me to add your name, please leave a comment on this PR.

### CODE REVIEW

**Comment by neutrinoceros** [comment]
Thanks for offering !
Here's my orcid number
0000-0001-8629-7068

**Review by chummels — APPROVED** [review]
LGTM.  Thanks for putting this together.

**Comment by neutrinoceros** [comment]
Oh btw it would be good to mention which versions of Python are officially supported (e.g. for which we're going to publish wheels). Right now it should be 3.7-3.10 (3.10 is a special case though because we have yet to publish wheels for yt itself, but it's supposed to happen soon)

**Comment by brittonsmith** [comment]
> Oh btw it would be good to mention which versions of Python are officially supported (e.g. for which we're going to publish wheels). Right now it should be 3.7-3.10 (3.10 is a special case though because we have yet to publish wheels for yt itself, but it's supposed to happen soon)

Oh, yes, good suggestion! I'll add this.

**Comment by brittonsmith** [comment]
There are just a couple things left to work out and @neutrinoceros and I are occupied at the moment. We will aim for a release early next week.

**Comment by neutrinoceros** [comment]
Found @cphyc's orcid too [0000-0003-2285-0332](https://orcid.org/0000-0003-2285-0332)

**Comment by willsonho2000** [comment]
Thanks a lot!
My orcid: 0000-0002-2316-1371

File: doc/source/changelog.rst [hunk 1]
```diff
@@ -9,13 +9,59 @@ Contributors
...
+
+     * the interface to the Rockstar halo finder is now compatible with the latest version of Rockstar Galaxies `PR #55 <https://github.com/yt-project/yt_astro_analysis/pull/55>`__
+     * all halo finders now support being run with time-series of datasets
+     * halo particle ids now savable with FoF and HOP halo finders `PR #52 <https://github.com/yt-project/yt_astro_analysis/pull/52>`__
+     * looping over halos is done with io chunks instead of ds.all_data for a significant speedup and reduction in memory
+     * Allow more flexibility for specifying rockstar particle mass `PR #84 <https://github.com/yt-project/yt_astro_analysis/pull/84>`__
+     * Add restart option for rockstar `PR #82 <https://github.com/yt-project/yt_astro_analysis/pull/82>`__
+     * Adding an outer_radius parameter to the iterative COM callback `PR #34 <https://github.com/yt-project/yt_astro_analysis/pull/34>`__
+
+ * Remove the sunyaev_zeldovich analysis module. This is now `pyxsim <http://hea-www.cfa.harvard.edu/~jzuhone/pyxsim/>`__. `PR #79 <https://github.com/yt-project/yt_astro_analysis/pull/79>`__
```
Comment (jzuhone) [inline doc/source/changelog.rst:36]: This has not been moved to pyxsim but to https://github.com/jzuhone/ytsz.

Comment (neutrinoceros) [reply doc/source/changelog.rst]: thanks John !

**Pull request merged**